projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -129,6 +129,9 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
     @ViewChild('inputGroup', { read: ElementRef, static: false })
     protected inputGroup: ElementRef;
 
+    @ViewChild('dropDownTarget', { read: ElementRef, static: false })
+    protected inputGroupDate: ElementRef;
```

Comment:
You could try to re-use the `inputGroup` element ref if you put the same id on the datepicker's input group.